content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,296 @@
+toc: true
+---
+
+This tutorial guides you through the process of creating a project game release layout for Windows PCs in **Open 3D Engine (O3DE)**. A *project game release layout* is a directory structure that contains the **Game Launcher** and the packaged assets needed to run the Game Launcher outside of the developer environment. 
+
+A release build requires *Bundled content*, product assets pulled from the project's `Cache/pc` directory into package files (`.pak`). The Game Launcher loads the bundled content that make up a project, such as its levels, objects, environment, and gameplay logic.
+
+The instructions here guide you through the following steps:
+- Set the starting level.
+- Process your project's assets.
+- Bundle your project's assets.
+- Create a project game release layout.
+- Run your project's Game Launcher from the release.
+
+
+## Prerequisites
+
+The following instructions assume that you have:
+
+- Set up O3DE on your computer as either a source engine or pre-built SDK engine. For help, refer to [Set up Open 3D Engine](/docs/welcome-guide/setup/). 
+
+- Created an O3DE project that contains at least one level. You may need to resolve any errors in your project to create a project game release layout.
+
+- Generated a Visual Studio project for either of the following:
+  
+    - If you are using a source engine, you must generate a Visual Studio project for your project. For help, refer to [Create a Visual Studio project](/docs/welcome-guide/create/creating-projects-using-cli/#create-a-visual-studio-project).
+  
+    - If you are using a pre-built SDK, you must generate a Visual Studio project for your engine. For help, refer to the **Pre-built SDK engine** tab in [Setup from GitHub](https://o3de.org/docs/welcome-guide/setup/setup-from-github/).
+
+- Registered your project to your engine. For help, refer to 
+
+
+This tutorial uses the following directories in the examples.
+
+- Project name and directory: `C:/MyProject`
+  - Project's profile build directory: `C:/MyProject/build/windows_vs2019/bin/profile`
+  - Project's release build directory: `C:/MyProject/build/windows_vs2019/bin/release`
+
+- Third-party directory: `C:/o3de-packages`
+- Source engine: `C:/o3de`
+- Pre-built SDK engine: `C:/o3de-install`
+
+
+## Set the starting level
+
+Before you create a project game release layout, specify the starting level that the Game Launcher will load first in your settings registry.
+
+1. Create an `autoexec.game.setreg` file in `MyProject/Registry` directory and add the following lines. This sets the starting level by using `Loadlevel` and providing a path to the level. The path must be relative to the `Cache/pc` directory. In this example, the level asset is `mainmenu.spawnable`.
+
+
+```
+{
+    "O3DE": {
+        "Autoexec": {
+            "ConsoleCommands": {
+                "LoadLevel": "levels/mainmenu/mainmenu.spawnable"
```

Comment:
After investigating the [code](https://github.com/o3de/o3de/blob/development/Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp#L221), this works with just the name of the level as well without any extensions
So the line  of `"LoadLevel": "mainmenu"` would work as well

Reply:
+1 for simpler version